pkg/operator/ceph/client/controller.go

```diff
@@ -74,7 +74,7 @@ func (c *ClientController) StartWatch(stopCh chan struct{}) {
 	}
 
 	logger.Infof("start watching client resources in namespace %q", c.namespace)
-	go k8sutil.WatchCR(ClientResource, c.namespace, resourceHandlerFuncs, c.context.RookClientset.CephV1().RESTClient(), &cephv1.CephClient{}, stopCh)
+	go k8sutil.WatchCR(ClientResource, c.namespace, resourceHandlerFuncs, c.context.RookClientset.CephV1().RESTClient(), &cephv1.CephClient{}, stopCh) //nolint
```

Comment:
You can change `WatchCR()` to not return an error. Looks like it always returns `nil` anyway, so no need for the error return.

Reply:
Done.